mrn_mysql_compat.h

```diff
@@ -694,7 +694,8 @@ typedef uint mrn_srid;
 #endif
 
 #ifdef MRN_MARIADB_P
-#  if (((MYSQL_VERSION_ID >= 101002) && (MYSQL_VERSION_ID < 101100)) || \
+#  if (((MYSQL_VERSION_ID >= 101102) && (MYSQL_VERSION_ID < 101200)) || \
+       ((MYSQL_VERSION_ID >= 101002) && (MYSQL_VERSION_ID < 101100)) || \
```

Comment:
How about removing `< 101100` check instead? We will not support MariaDB 10.11.1 because MariaDB 10.11.1 is RC not GA: https://mariadb.com/kb/en/changes-improvements-in-mariadb-1011/#list-of-all-mariadb-1011-releases

```suggestion
#  if (((MYSQL_VERSION_ID >= 101002) || \
```

Reply:
Hi @kou, your suggestion have an extra parenthesis and causes an error when compile.

Reply:
Oh, sorry.